### PR TITLE
[SMALLFIX] Fix storage space validation for ramfs

### PR DESCRIPTION
**Pull request by bf8086**
Currently running validateEnv command on linux with mounted ramfs for worker storage will fail because ramfs does not report correct space information. I am adding a check so we skip checking available space for ramfs.

### CODE REVIEW

**Comment by bf8086** [comment]
@yupeng9 @aaudiber PTAL. Thanks!

File: core/server/common/src/main/java/alluxio/cli/validation/Utils.java [hunk 1]
```diff
@@ -93,6 +98,30 @@ public static boolean isAlluxioRunning(String className) {
     return nodes;
   }
 
+  /**
+   * Checks whether a path is the mounting point of a RAM disk volume.
+   * @param path  a string represents the path to be checked
+   * @param fsTypes  an array of strings represents expected file system type
+   * @return true if the path is the mounting point of volume with one of the given fsTypes,
+   * false otherwise
```
Comment (aaudiber) [inline core/server/common/src/main/java/alluxio/cli/validation/Utils.java]: indent

File: core/server/common/src/main/java/alluxio/cli/validation/Utils.java [hunk 2]
```diff
@@ -93,6 +98,30 @@ public static boolean isAlluxioRunning(String className) {
     return nodes;
   }
 
+  /**
+   * Checks whether a path is the mounting point of a RAM disk volume.
+   * @param path  a string represents the path to be checked
```
Comment (aaudiber) [inline core/server/common/src/main/java/alluxio/cli/validation/Utils.java]: extra space

File: core/server/common/src/main/java/alluxio/cli/validation/Utils.java [hunk 3]
```diff
@@ -93,6 +98,30 @@ public static boolean isAlluxioRunning(String className) {
     return nodes;
   }
 
+  /**
+   * Checks whether a path is the mounting point of a RAM disk volume.
```
Comment (aaudiber) [inline core/server/common/src/main/java/alluxio/cli/validation/Utils.java]: blank line after 1-sentence description

File: core/server/common/src/main/java/alluxio/cli/validation/StorageSpaceValidationTask.java [hunk 4]
```diff
@@ -65,16 +65,25 @@ public boolean validate(Map<String, String> optionsMap) {
...
-          // skip checking if RAM disk is not mounted
           System.out.format("RAM disk is not mounted at %s, skip validation.%n", dirPaths[0]);
           continue;
         }
 
+        boolean hasRamfsLocation = false;
         for (int i = 0; i < dirPaths.length; i++) {
           int index = i >= dirQuotas.length ? dirQuotas.length - 1 : i;
+          if (Utils.isMountingPoint(dirPaths[i], new String[] {"ramfs"})) {
+            System.out.format("ramfs is mounted at %s which does not report space information,"
```
Comment (aaudiber) [inline core/server/common/src/main/java/alluxio/cli/validation/StorageSpaceValidationTask.java]: "ramfs mounted at %s does not report space information"

**Pull request merged**